themes-default/slim/static/js/router.js

```diff
@@ -11,6 +12,14 @@ if (!window.router) {
         base: document.body.getAttribute('api-root').replace('api/v2/', ''),
         mode: 'history',
         routes: Object.assign(routes || [], [{
+            path: '/login',
+            name: 'login',
+            meta: {
+                title: 'Login',
+                header: 'Medusa'
```

Comment:
No header for the login page. It has its own header.
It's not currently an issue, because it doesn't use the VueRouter template (`index.mako`),
but it will be when it does.